fjord/analytics/views.py

```diff
@@ -235,16 +227,16 @@ def unknown_to_empty(text):
     visible_products.append('')
 
     if search_product in visible_products:
-        f &= F(product=unknown_to_empty(search_product))
+        f &= F('term', product=unknown_to_empty(search_product))
         current_search['product'] = search_product
 
         if search_version is not None:
             # Note: We only filter on version if we're filtering on
             # product.
-            f &= F(version=unknown_to_empty(search_version))
+            f &= F('term', version=unknown_to_empty(search_version))
             current_search['version'] = search_version
     else:
-        f &= F(product__in=visible_products)
```

Comment:
Is this right? Before you were doing a `__in` query, but now you are doing a normal terms query.